src/style.css

```
@@ -0,0 +1,38 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir.ttc);
+}
+
+.serch {

```

```suggestion
.search {
```
Did you mean `search`?

---

yes